src/PostgREST/PgQuery.hs

```diff
@@ -241,7 +241,7 @@ pgFmtJsonbPath :: T.Text -> T.Text
 pgFmtJsonbPath p =
   pgFmtJsonbPath' $ fromMaybe (ColIdentifier p) (parseJsonbPath p)
   where
-    pgFmtJsonbPath' (ColIdentifier i) = pgFmtIdent i
+    pgFmtJsonbPath' (ColIdentifier i) = i
```

Comment:
I like the sound of the second option because it means everything happens in a safe uniform way. If you make that change we can review and hopefully get this merged.